lighthouse-core/config/constants.js

```diff
@@ -51,6 +51,7 @@ const defaultSettings = {
   disableStorageReset: false,
   disableDeviceEmulation: false,
   emulatedFormFactor: 'mobile',
+  channel: 'node',
```

Comment:
and can assert this in `lighthouse-core/test/index-test.js` (basically just copy `'should return formatted LHR when given no categories'` and delete most of it :)